xls/examples/BUILD

```diff
@@ -32,6 +32,7 @@ load(
 load("@rules_hdl//verilog:providers.bzl", "verilog_library")
 load("@rules_hdl//synthesis:build_defs.bzl", "synthesize_rtl")
 load("@rules_hdl//place_and_route:build_defs.bzl", "place_and_route")
+load("@rules_hdl//gds_write:build_defs.bzl", "gds_write")
```

Comment:
nit: Load statement is out of its lexicographical order.

Reply:
fixed